protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/processinstance/StartInstruction.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.protocol.impl.record.value.processinstance;
+
+import io.camunda.zeebe.msgpack.property.EnumProperty;
+import io.camunda.zeebe.msgpack.property.StringProperty;
+import io.camunda.zeebe.msgpack.value.ObjectValue;
+import io.camunda.zeebe.util.buffer.BufferUtil;
+
+public class StartInstruction extends ObjectValue {
```

Comment:
:wrench: We could extend the class name to give more context. `CreationStartInstruction` or `ProcessInstanceCreationStartInstruction` could be more concrete.